jsapp/js/formbuild/containers/KoboMatrix.es6

```diff
@@ -96,17 +96,17 @@ class KoboMatrix extends React.Component {
 
   autoName(val, type, ln = false) {
     var names = [];
-    const kobomatrix_list = this.state.kobomatrix_list;
     var data = this.state.data;
 
     if (type === 'column') {
-      data.get('cols').forEach(function(ch){
+      data.get('cols').forEach(function (ch) {
         names.push(data.getIn([ch, '$autoname']));
       });
     } else {
-      data.get('choices').forEach(function(ch){
-        if (ch.get('list_name') == ln)
+      data.get('choices').forEach(function (ch) {
```

Comment:
Should `survey`, `choices`, `settings`, `list_name`, `$autoname`, `$autovalue` etc. also be things we can include with the new `createEnum()` function?

Reply:
I would say: "maybe" :D If we'd write more new code that uses some of these names, then we would use an enum, but as this is just a bugfix with some code style cleanup, I would leave it